.github/workflows/testsuite.yml

```diff
@@ -70,17 +58,17 @@ jobs:
         fi
       shell: bash
     - name: Install extra dependencies
-      if: ${{ ! matrix.experimental }}
+      if: ${{ python-version == "3.12-dev" }}
```

Comment:
Shoudln't that be != ?

Reply:
Of course 🤦🏻

Reply:
The string literal must be written using single quotes: https://docs.github.com/en/actions/learn-github-actions/expressions#literals